change/@ni-nimble-components-5ec786bd-2a49-43a5-a397-4dcc3d8eaed3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mollykreis would you mind buddying this PR?

Reply:
@mollykreis I'm gonna merge this PR so we can get some runtime on it but will address concerns in a follow-up